### ISSUE TITLE
Infinite indexing

### ISSUE THREAD

**Issue by ktrzos** [reasonml-editor/reasonml-idea-plugin#214]
When uninstalling this plugin, everything works fine. But with it, I have infinite project indexing.

plugin version: 0.87-2019.3

![Selection_001](https://user-images.githubusercontent.com/14212721/77639036-5c072a00-6f58-11ea-9db1-81119dd2feca.png)

Edit: works fine on version `0.86.1-2019.3 → 0.87-2019.3`

**Comment by giraud** [comment]
what is the bsconfig.json ?

**Comment by Coobaha** [comment]
It should be the same issue that I’ve reported to you in discord: gson parsing error because of json5 format

**Comment by ktrzos** [comment]
@giraud 

```json
{
  "name": "Project",
  "sources": [
    {
      "dir": "src",
      "subdirs": true
    },
    {
      "dir": "__tests__",
      "subdirs": true,
      "type": "dev"
    }
  ],
  "package-specs": {
    "module": "commonjs",
    "in-source": true
  },
  "suffix": ".bs.js",
  "bs-dependencies": [
    "reason-react",
    "@glennsl/bs-json",
    "bs-webapi",
    "bs-fetch",
    "bs-moment",
    "reason-date-fns",
    "re-classnames",
    "reason-apollo",
    "reason-apollo-hooks",
    "re-debouncer",
    "bs-reform",
    "reschema"
  ],
  "bs-dev-dependencies": [
    "@glennsl/bs-jest",
    "bs-react-test-renderer"
  ],
  "ppx-flags": [
    "graphql_ppx/ppx",
    "lenses-ppx/ppx"
  ],
  "reason": {
    "react-jsx": 3
  },
  "refmt": 3,
  // "bsc-flags": ["-bs-g"],
  "generate-merlin": true,
  "warnings": {
    "number": "-45"
  }
}

```

**Comment by giraud** [comment]
should be fixed in 0.87.1
